.github/ISSUE_TEMPLATE.md

```diff
@@ -1,42 +1,47 @@
 <!--
-Please have a look at the issue templates you get when you click "New issue" in the GitHub UI.
-We very much prefer issues created by using one of these templates.
+Please select an appropriate issue template when you click "New issue" in the GitHub UI.
+We would appreciate it if you could use one of the provided issue templates. Thank you!
 -->
 
 ### Issue Type
-<!-- Please check the boxes by putting an x in the [ ] like so: [x] -->
+<!-- Please check the boxes by putting an `x` in the `[ ]` like so: `[x]` -->
 
 - [ ] Bug Report
 - [ ] Feature Request
 - [ ] Support Question
 
 ## Description
+<!-- Briefly describe an issue. -->
+
+
 
 ## Information
-<!-- Include all relevant information that might help understand and reproduce the problem -->
+<!-- Include all related information that might help understand and reproduce the problem. -->
 
-### Command or Code
-<!-- What command or code is needed to reproduce the problem? -->
 
-### Environment, Platform, Device
-<!-- In what environment, on what platform or on which device are you experiencing the issue? -->
+
+### Command or Code
+<!-- Describe a use case(s) that causes this problem. What command or code reproduces the problem? -->
 
 
 
-### Version information
+### Environment and Versions
 <!-- 
-What are relevant versions you are using?
+Describe your environment, platforms, devices and Operating System. 
+Include all versions of Cordova tooling, Operating System, also Third-Party tools and programs.
+
 For example:
-Cordova: Cordova CLI, Cordova Platforms, Cordova Plugins 
-Other Frameworks: Ionic Framework and CLI version
-Operating System, Android Studio, Xcode etc.
+Cordova: Cordova CLI v0.0.0, Cordova Platform v0.0.0, Cordova Plugin v0.0.0
+Other Tools/Frameworks: Ionic Framework 0.0.0 and Ionic CLI v0.0.0
+Operating System, Android Studio, Xcode, etc.
 -->
 
 
 
 ## Checklist
 <!-- Please check the boxes by putting an `x` in the `[ ]` like so: `[x]` -->
 
-- [ ] I searched for already existing GitHub issues about this
-- [ ] I updated all Cordova tooling to their most recent version
-- [ ] I included all the necessary information above
+**Before submitting an issue, make sure to:**
```

Comment:
Why the switch from making the issues sound like the issue creator is telling something, to here the issue template talking to them?

Reply:
To me, a checklist means a list of tasks to be completed. So the name itself to me implies that this checklist is oriented to the issue creator. Plus, I believe, psychologically it adds more stress and significance on the checklist items that they all must be checked before submitting an issue.